packages/at_secondary_server/lib/src/connection/outbound/outbound_client.dart

```diff
@@ -67,8 +68,12 @@ class OutboundClient {
   /// Throws a [SecondaryNotFoundException] if secondary is url is not found for atsign
   /// Throws a [SocketException] when a socket connection to secondary cannot be established
   /// Throws a [HandShakeException] for any exception in the handshake process
-  Future<bool> connect({bool handshake = true}) async {
-    logger.finer('connect(handshake:$handshake) called for $toAtSign');
+  Future<bool> connect() async {
```

Comment:
Removed the `handshake` parameter since OutboundClients are always created for either unauthenticated or pol-authenticated connections, but not both,. Thus, handshakeRequired is now a final field in the class, required in constructor.